spec/models/question_spec.rb

```diff
@@ -2,17 +2,40 @@
 #
 # Table name: questions
 #
-#  id         :integer          not null, primary key
-#  title      :string(255)
-#  body       :text(65535)
-#  user_id    :integer
-#  created_at :datetime         not null
-#  updated_at :datetime         not null
-#  sum_votes  :integer          default(0)
+#  id                 :integer          not null, primary key
+#  title              :string(255)
+#  body               :text(65535)
+#  user_id            :integer
+#  created_at         :datetime         not null
+#  updated_at         :datetime         not null
+#  sum_votes          :integer          default(0)
+#  approved_answer_id :integer
 #
 
 require 'rails_helper'
 
 RSpec.describe Question, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+  it "有効なファクトリをもつこと" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.